app/views/idv/session_errors/_in_person_proofing_options.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if IdentityConfig.store.in_person_proofing_enabled %>
+  <%= render TroubleshootingOptionsComponent.new(new_features: true) do |c| %>
+    <% c.header(heading_level: :h2).with_content(t('idv.troubleshooting.headings.are_you_near')) %>
```

Comment:
`:h2` is the default heading level, so I expect this could be simplified to...

```suggestion
    <% c.header.with_content(t('idv.troubleshooting.headings.are_you_near')) %>
```

Reply:
I ended up having to use the block syntax in 07ba114b2ec978723c96b78b2f41a6ed6b5b3cd9 because `c.header.with_content` raise a NoMethodError :/